app/src/main/java/de/tum/in/tumcampusapp/component/tumui/tutionfees/TuitionNotificationScheduler.kt

```diff
@@ -0,0 +1,54 @@
+package de.tum.`in`.tumcampusapp.component.tumui.tutionfees
+
+import de.tum.`in`.tumcampusapp.component.tumui.tutionfees.model.Tuition
+import org.joda.time.DateTime
+import org.joda.time.Days
+
+/**
+ * This class is responsible for scheduling reminders about remaining tuition fees. It first reminds
+ * the user 30 days before the deadline and repeats those reminders more frequently as the deadline
+ * comes closer.
+ */
+object TuitionNotificationScheduler {
+
+    /**
+     * Returns the timestamp of when the next reminder about tuition fees should be scheduled. It
+     * takes into account the remaining time until the tuition fees deadline.
+     *
+     * @param tuition The [Tuition] with its deadline
+     * @return The timestamp of the next reminder in milliseconds
+     */
+    fun getNextNotificationTime(tuition: Tuition): Long {
+        val notificationTimes = arrayOf(
+                Days.days(30),
+                Days.days(14),
+                Days.days(7),
+                Days.days(4),
+                Days.days(3),
+                Days.days(2),
+                Days.days(1),
+                Days.days(0)
```

Comment:
please document the reasoning behind those days

Reply:
Done.